src/app.js

```diff
@@ -14,7 +14,7 @@ const persistConfig = {
   throttle: 1000,
   key: getAppPath(),
   storage,
-  whitelist: ['global', 'user'],
```

Comment:
We should consider preserving the `user` namespace here as the authentication flow involves caching user credentials and then eventually confirming their validity against the server.

Reply:
@gurbirkalsi if we preserve the `user` namespace, the state of redux for user will not getting cleared even after refresh and it is automatically taking up the value from the previous login. That is, if the user doesn't login and go back to the PrivateRoute, it will still take the username of the previously logged in user. Is it an expected behaviour for log in system?

Reply:
We should have a `logout` Redux action that takes the `user` namespace and clears all authentication related fields from the namespace state. Eventually, we will confirm the validity of the cached user token against the server which will then be deleted or preserved (via Redux actions) depending on the result of the confirmation.

In addition, we should consider creating an `auth` namespace as the `user` namespace houses global state for user preferences/favorites that we should avoid mixing with `auth`.

Reply:
I have created different namespaces for different auth and user data.